packages/lib/membership/service.ts

```diff
@@ -121,9 +139,15 @@ export const deleteMembership = async (userId: string, teamId: string): Promise<
   return deletedMembership;
 };
 
-export const transferOwnership = async (currentOwnerId: string, newOwnerId: string, teamId: string) => {
+export const transferOwnership = async (
+  currentOwnerId: string,
+  newOwnerId: string,
+  teamId: string
+): Promise<TMembership[]> => {
+  validateInputs([currentOwnerId, ZString], [newOwnerId, ZString], [teamId, ZString]);
+
   try {
-    await prisma.$transaction([
+    const owners = await prisma.$transaction([
```

Comment:
`owners` isn't the fitting name as two `memberships` are returned

Reply:
Yeah I named it this way because of the name of the service `transferOwnership`.

Should I rename the function to `transferMembership` in addition to renaming `owners`?

Reply:
I think transferOwnership describes better what happens in the service.
But I would choose `memberships` for the variable name as `owners` isn't correct as there is only one owner an with `owner` I would assume it would be a user and not a membership.

Reply:
ahh I see, sounds good. 👍

Reply:
Done